### ISSUE TITLE
Reword work shift reminder emails to make arrival time more clear

### ISSUE THREAD

**Issue by smoyth** [gather-community/gather#479]
From Syndallas:

"The reminders are good. Thank you.  But, maybe the reminder for the assistant cook shift should be reworded so it is clear the worker should arrive at 4pm rather than 6:15pm.

This announcement makes it sound like the worker should arrive at 6:15pm"

This is a reminder that you are scheduled as Assistant Cook for a meal by Jennifer Janssen on Tue Dec 1, 6:15pm at Great Oak Common House.

Your help is crucial! Please arrive promptly for your shift!

I don't know what time the TS and SW workers should show up, but at GO it has been 4pm since the dawn of our meals program.

_Originally created by **Willie Northway** at **2015-12-01 03:45**, migrated from [redmine-#3698](https://redmine.sassafras.coop/issues/3698)_

**Comment by smoyth** [comment]
On Mon, Feb 15, 2016 at 9:14 AM, Robby Griswold
[robbygriswoldmail@gmail.com](mailto:robbygriswoldmail@gmail.com) wrote:  
hey there,

I'm looking and looking and can't find the description that tells me what time
I need to report to the CH Kitchen for my assistant cook shift tomorrow. Could
someone tell me?

_By **Willie Northway** at **2016-02-17 08:25**_

**Comment by smoyth** [comment]
We've had a request from TS to update this too. I'm going to implement it as
an optional "additional text" field that can be set per-community. I think we
should still publish the serving time of the meal, since it may not be the
same for all communities or meals (we have morning brunches).

Just waiting on Deb for some text and then I'll get it done, should be next
week.

_By **Tom Smyth** at **2016-03-03 07:45**_

**Comment by smoyth** [comment]
* **Priority** changed from _4_ to _3_

Katie has just commented on this as well:

"This says my assistant cook shift is at 6:15 - it should say that it is from
4-6:15. Of course, I know the hours of an assistant cook shift - but new
members might not.

Ironically, the email says, "Please arrive promptly for your shift!". But then
doesn't give an accurate start time."

_By **Willie Northway** at **2016-06-26 19:33**_

**Comment by smoyth** [comment]
* **Priority** changed from _3_ to _2_

From Tammy:  
4:13pm He (she?) is one of my assistant cooks tonight and hasn't arrived. I
don't  
know how to contact them, anyone have a phone number? Thanks.

4:30pm No answer, and the name on the voice mail is someone other than Gaz.  
Might need some help setting tables later, if anyone is available..."

From Mary King:  
"Gaz showed up for his cook shift tonight...at 6:15pm! That's because the  
email reminder he received listed the time for his shift as 6:15. He was  
bummed because he was excited to work his first shift."

I suggest just doing the math and telling assistant cooks to show up 2 hours
15 minutes before, and cleaners 45 minutes after the dinner is served. I'm
happy to come up with some language if necessary.

_By **Willie Northway** at **2016-09-07 04:15**_

**Comment by smoyth** [comment]
* **Status** changed from _New_ to _Cancelled_

This is now a duplicate of [#5067](https://redmine.sassafras.coop/issues/5067
"Feature: Per-meal timing adjustments \(New\)"). I want to kill multiple birds
with one stone here. Sometimes folks are forgetting to reserve both kitchen
and dining room, you've had issues with the auto dining room booking being
overzealous, then the shift time thing, they are all related. See 5067 for
what I mean. So cancelling this one.

_By **Tom Smyth** at **2016-09-08 06:04**_
